core/src/env2/error.rs

```diff
@@ -40,13 +40,13 @@ pub enum Error {
     Call(CallError),
     Create(CreateError),
     Codec(scale::Error),
-    InvalidStorageKey,
-    InvalidStorageRead,
-    InvalidContractCall,
-    InvalidContractCallReturn,
-    InvalidContractInstantiation,
-    InvalidContractInstantiationReturn,
-    InvalidRandomSeed,
+    #[from(ignore)] InvalidStorageKey,
+    #[from(ignore)] InvalidStorageRead,
+    #[from(ignore)] InvalidContractCall,
+    #[from(ignore)] InvalidContractCallReturn,
+    #[from(ignore)] InvalidContractInstantiation,
+    #[from(ignore)] InvalidContractInstantiationReturn,
+    #[from(ignore)] InvalidRandomSeed,
```

Comment:
Note: Already opened [an issue](https://github.com/JelteF/derive_more/issues/105) for this abomination.